langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockAnthropicCompletionChatModel.java

```diff
@@ -63,4 +69,10 @@ public enum Types {
             this.value = modelID;
         }
     }
+
+    @Override
+    public Response<AiMessage> generate(List<ChatMessage> messages) {
```

Comment:
What about other `generate` methods?

Reply:
done for all methods

Reply:
OK, it does not make sense for `BedrockAnthropicMessageChatModel` either 😅 Sorry